packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -12,6 +12,38 @@ model MultiPartRequest {
   profileImage: bytes;
 }
 
+model Address {
+  city: string;
+}
+
+model ComplexPartsRequest {
+  id: string;
+  address: Address;
+  profileImage: bytes;
+  previousAddresses: Address[];
+  pictures: bytes[];
+}
+
+model JsonPartRequest {
+  address: Address;
+  profileImage: bytes;
+}
+
+model BinaryArrayPartsRequest {
+  id: string;
+  pictures: bytes[];
```

Comment:
do we have a pure `bytes` case?

Reply:
Yeah, it has been covered by `multiBinaryParts`